app/furniture/marketplace.rb

```diff
@@ -3,7 +3,15 @@ class Marketplace
 
   def self.append_routes(router)
     router.resource :marketplace, only: [] do
-      router.resources :products, only: %i[create index], module: "marketplace"
+      router.resources :products, only: %i[create index], module: 'marketplace'
     end
   end
+
+  def products
+    Marketplace::Product.where(space: space)
+  end
+
+  def to_partial_path
```

Comment:
Maybe add a comment that this implements https://apidock.com/rails/v6.1.3.1/ActiveModel/Conversion/to_partial_path , for clarity (I had to look up what was this doing, I wasn't familiar with this method, as I've never had the need to implement this myself).

Reply:
I like it!